cfgmgr/buffermgr.cpp

```diff
@@ -391,6 +393,11 @@ void BufferMgr::doPortQosTableTask(Consumer &consumer)
     {
         KeyOpFieldsValuesTuple tuple = it->second;
         string port_name = kfvKey(tuple);
+        if (port_name == PORT_NAME_GLOBAL)
+        {
+            // Ignore the entry for global level 
+            continue;
```

Comment:
Better to erase `it` from `m_toSync` otherwise this sentence will be hit every time.

Reply:
Good catch. Thanks